docs/integrations/databases/opentelemetry/redis.md

```diff
@@ -0,0 +1,135 @@
+import TabItem from '@theme/TabItem';
+
+<img src={useBaseUrl('img/integrations/databases/redis.png')} alt="Thumbnail icon" width="50"/> <img src={useBaseUrl('img/send-data/otel-color.svg')} alt="Thumbnail icon" width="45"/>
+
+[Redis](https://redis.io/docs/about/) is an in-memory data structure that implements a distributed, in-memory key-value database with durability options.
+
+The Sumo Logic App for Redis helps you monitor the Redis database cluster. The preconfigured dashboards provide detailed analysis based on logs. The panels provide details such as RDBMemory Usage, events, RDB, and AOF events.
+
+Redis logs are sent to Sumo Logic through OpenTelemetry [filelog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver).
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Redis-OpenTelemetry/schematics-redis.png')} alt="Redis Schematics" />
+
+## Log Types
+
+This app supports logs for Redis in Cluster mode or Standalone mode. The Redis logs are generated in files as configured in the cluster node configuration files in `/etc/redis/redis.conf`. For more details on Redis logs, [click here](https://redislabs.com/ebook/part-2-core-concepts/chapter-5-using-redis-for-application-support/5-1-logging-to-redis/).
+
+## Creating Fields in Sumo Logic for Redis
+
+The following are [Fields](https://help.sumologic.com/docs/manage/fields/) that will be created as part of the Redis App install if not already present.
+
+* **`db.cluster.name`** - User configured. Enter a name to identify this Redis cluster. This cluster name will be shown in the Sumo Logic dashboards.
+* **`db.system`** - Has fixed value of redis.
+* **`deployment.environment`** - User configured. This is the deployment environment where the Redis cluster resides. For example: dev, prod or qa.
+* **`sumo.datasource`** - has fixed value of redis.
+
+## Prerequisite
+
+This section provides instructions for configuring log collection for Redis running on a non-Kubernetes environment. By default, Redis logs are stored in a log file.
+
+Follow the instructions to set up log collection:
+
+1. To configure the Redis log file, locate your local `[redis.conf](https://download.redis.io/redis-stable/redis.conf)` configuration file in the database directory. By default, Redis logs are stored in `/var/log/redis/redis-server.log`.
+1. After determining the location of conf file, open your `redis.conf` configuration file in a text editor to modify its logging parameters as such:
+   1. Specify the server verbosity level. The value `loglevel` in conf file can be set to one of the following:
+      * debug (a lot of information, useful for development/testing)
+      * verbose (many rarely useful info, but not a mess like the debug level)
+      * notice (moderately verbose, what you want in production probably)
+      * warning (only very important/critical messages are logged) loglevel notice
```

Comment:
@angadrandhawa1 @sumoanema @arunpatyal  @stacykor , since we're writing for a developer audience, I don't think we have to define these. Let me know what you think.